main.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+function daysInYear(int) {
+  const year = int;
+
+  if (year % 1 === 0 && typeof year === 'number') {
```

Comment:
It will be more logic to swap expression
`typeof year === 'number' && year % 1 === 0`
also, Number have method `isInteger`